concrete/single_pages/dashboard/system/basics/accessibility.php

```diff
@@ -2,47 +2,51 @@
     <?php
     Core::make('token')->output('accessibility');
     ?>
-    <div class="checkbox">
-        <label>
-            <input name="show_titles" value="1" type="checkbox" <?= $show_titles ? 'checked' : '' ?> />
-            <?= t('Enable Toolbar Titles') ?>
-        </label>
-    </div>
-    <div class="checkbox">
-        <label>
-            <input name="show_tooltips" value="1" type="checkbox" <?= $show_tooltips ? 'checked' : '' ?> />
-            <?= t('Enable Prominent Toolbar Tooltips') ?>
-        </label>
-    </div>
-    <div class="checkbox">
-        <label>
-            <input name="increase_font_size" value="1" type="checkbox" <?= $increase_font_size ? 'checked' : '' ?> />
-            <?= t('Increase Toolbar Font Size') ?>
-        </label>
-    </div>
-
-    <hr/>
-
     <fieldset>
-        <label class="control-label"><?=t('File Manager')?></label>
-        <div class="checkbox">
-            <label>
-                <input name="full_lisiting_thumbnails" value="1" type="checkbox" <?= $full_lisiting_thumbnails ? 'checked' : '' ?> />
-                <?= t('Enable Full Size Image Thumbnails') ?>
-            </label>
+        <div class="form-group">
+            <div class="form-check">
+                <label>
+                    <input class="form-check-input" name="show_titles" value="1" type="checkbox" <?= $show_titles ? 'checked' : '' ?> />
+                    <?= t('Enable Toolbar Titles') ?>
```

Comment:
I think bootstrap 4 form check classes require the use of a form-check-label class around the labels, and they don't want the <label> class to wrap around the input.

Reply:
Done